google/cloud/aiplatform/training_jobs.py

```diff
@@ -1445,6 +1485,19 @@ def _wait_callback(self):
 
             self._has_logged_custom_job = True
 
+        if self._has_logged_custom_job and self._gca_resource.training_task_inputs.get(
+            "enable_web_access"
+        ):
+            if not self._custom_job:
+                custom_job_resource_name = self._gca_resource.training_task_metadata.get(
+                    "backingCustomJob"
+                )
+                self._custom_job = jobs.CustomJob.get(
+                    resource_name=custom_job_resource_name
+                )
+            self._web_access_uris = self._custom_job.web_access_uris
+            self._log_web_access_uris()
```

Comment:
Perhaps we can pass in `web_access_uris` so we don't need to maintain `self._web_access_uris` property.